client/me/controller.js

```diff
@@ -281,7 +281,7 @@ export default {
 		if ( transactionId ) {
 			analytics.pageView.record( basePath + '/receipt', ANALYTICS_PAGE_TITLE + ' > Billing History > Receipt' );
 
-			ReactDom.render(
+			context.render(
```

Comment:
we could call it this way:
```js
const reactElement = React.createElement( ViewReceiptModal, { transaction: billingData.getTransaction( transactionId ) } );
renderWithProvider( reactElement, 'tertiary', context.store );
```
I guess we always use `document.getElementById` method, so we could simplify also that part.